ansible/playbooks/roles/rook/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+rook_helm_chart_file_name: rook-ceph-v1.8.1.tgz
+rook_chart_namespace: rook-ceph
+rook_helm_chart_name: rook-ceph
+rook_enabled: true
```

Comment:
Should this var be in defaults with no ability to overwrite for user?

Reply:
The idea from @to-bar is to enable rook via feature-mapping

Reply:
This was temporary.